src/docfx/lib/git/GitRepoInfoProvider.cs

```diff
@@ -84,6 +84,26 @@ public GitRepoInfoProvider(Docset docset)
             return (errors, ToGitUserInfo(authorInfo), contributors.Select(ToGitUserInfo).ToArray(), updatedAt);
         }
 
+        public string GetEditLink(Document document)
+        {
+            Debug.Assert(document != null);
```

Comment:
How about using a string template for edit link? https://github.com/dotnet/docfx/issues/2941

Reply:
will do some refactor when supporting git host other than GitHub